src/goom_config.h

```diff
@@ -2,10 +2,10 @@
 #define GOOM_CONFIG
 
 #ifdef MID
-#define GOOM_TEXTURE_WIDTH  256
+#define GOOM_TEXTURE_WIDTH  455
```

Comment:
Peculiar sizes. 512x1024 is better for hw. Also it makes sense to out this as an option as this is software rendered and can eat some cpu (bot expecting you to do that bit)